components/calendar_components/TaskList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Task from "./Task";
+
+export default function TaskList({ tasks, toggleTask }) {
+  return (
+    <div className="task-list">
+      <div className="lefttasks">
+        {tasks.filter((task) => !task.complete).length} Aufgaben übrig
```

Comment:
I would recommend putting this operation before the return statement and saving it in an menaingful variable. Just the filter function here is not good readable.
 ```js
const numberOfUncompletedTask = tasks.filter((task) => !task.complete).length
```